src/pages/javascript/standard-objects/array/array-prototype-splice/index.md

```diff
@@ -3,13 +3,59 @@ title: Array.prototype.splice
 ---
 ## Array.prototype.splice
 
-The  `splice()` method removes and/or adds elements within an array.
+The splice method is similar to <a href='https://guide.freecodecamp.org/javascript/standard-objects/array/array-prototype-slice' target='_blank' rel='nofollow'>Array.prototype.slice</a>, but unlike `slice()` it mutates the array it is called on. It also differs in that it can be used to add values to an array as well as remove them.
 
-## Example
-``` javascript
-var myArray = ['a', 'b', 'c', 'd'];
-letters.splice(4, 0, 'e'); // insert 'e' at 2-index
-// myArray is now ["a", "b", "c", "d", "e"]
+### Parameters
+`splice()` can take one or more
+#### splice(start)
+If only one parameter is included, then `splice(start)` will remove all array elements from `start` to the end of the array.
+```js
+let exampleArray = ['first', 'second', 'third', 'fourth'];
+exampleArray.splice(2);
+// exampleArray is now ['first', 'second'];
 ```
+
+If `start` is negative, it will count backwards from the end of the array.
+```js
+let exampleArray = ['first', 'second', 'third', 'fourth'];
+exampleArray.splice(-1);
+// exampleArray is now ['first', 'second', 'third'];
+```
+
+#### splice(start, deleteCount)
+If a second parameter is included, then `splice(start, deleteCount)` will remove `deleteCount` elements from the array, beginning with `start`.
+```js
+let exampleArray = ['first', 'second', 'third', 'fourth'];
+exampleArray.splice(-1, 2);
+// exampleArray is now ['first', 'fourth'];
```

Comment:
For this example, the output is `["first", "second", "third"]` because `2` is greater than `arr.length - start` where start is `3` so its removing till `arr.length`, its following this rule:
>if its value is larger than array.length - start (that is, if it is greater than the number of elements left in the array, starting at start), **then all of the elements from start through the end of the array will be deleted.**

Reply:
You're right. I had intended the example to be `splice(1, 2)` rather than `splice(-1, 2)`. Good catch.